src/bp/core/services/dialog/prompts/confirm.ts

```diff
@@ -0,0 +1,59 @@
+import { IO, Prompt, PromptConfig } from 'botpress/sdk'
+import * as sdk from 'botpress/sdk'
+import { extractEventCommonArgs } from 'common/action'
+import yn from 'yn'
+
+class PromptConfirm implements Prompt {
+  public static config: PromptConfig = {
+    type: 'confirm',
+    label: 'Confirm',
+    valueType: 'boolean',
+    minConfidence: 1,
+    noValidation: true,
+    params: {
+      question: { label: 'Question to ask to user', type: 'string' }
+    }
+  }
+
+  private _question: string
+
+  constructor({ question }) {
+    this._question = question
+  }
+
+  extraction(event: IO.IncomingEvent) {
+    const yesOrNo = yn(event.payload?.payload || event.preview)
```

Comment:
hmmm I'm not sure this works. 

I mean this will work in english only. and even if this we were taking only english into account then confirmations like 

"correct" and "yea" and "agreed!" won't work.

Reply:
Right now the prompt shows yes/no buttons, which are correctly handled. The ideal solution would be to have built in intents to accept or decline

Reply:
Yes, well how do you see that ? 

each prompt will come with intent and entity ?

Reply:
I think we could create default intents which handle multiple language (or make it easy to import them in a new bot). Having some intents by default could be useful (for example, cancel)